packages/flutter_tools/templates/plugin/lib/projectName_method_channel.dart.tmpl

```diff
@@ -0,0 +1,19 @@
+import 'dart:async';
```

Comment:
Is this actually being used?

Reply:
Don't you need it for Future?

Reply:
Simple `Future` usage is vended by `dart:core` starting in Dart 2.1.

Reply:
Nope, I'm wrong about that too. Removed.